9c-main/multiplanetary/network/9c-network.yaml

```
@@ -402,4 +402,27 @@ acc:
 
 volumeRotator:
   enabled: true
-  nodeSelectorValue: "9c-main-r7g_xl_2c"
\ No newline at end of file
+  nodeSelectorValue: "9c-main-r7g_xl_2c"
+
+rudolfService:
+  enabled: true
+
+  config:
+    graphqlEndpoint: "http://9c-main-rpc-1.nine-chronicles.com/graphql"
```

```suggestion
    graphqlEndpoint: "http://9c-main-full-state.nine-chronicles.com/graphql"
```

Could you use the full-state node instead? 🙏